saskatoon/member/models.py

```diff
@@ -103,6 +103,16 @@ def roles(self):
         ''' lists user's role names'''
         return [dict(AUTH_GROUPS).get(g.name) for g in self.role_groups]
 
+    @property
+    def is_onboarding(self):
+        ''' return if user has yet to go through onboarding flow (only volunteer role with password)'''
+        # Retrieve only name fields from QuerySet of Groups
+        group_names = [g.name for g in self.role_groups]
+        is_volunteer = 'volunteer' in group_names
+        is_pickleader = 'pickleader' in group_names
+        has_password = self.password != ''
+        return is_volunteer and has_password and not is_pickleader
```

Comment:
nit: `is not pickleader` should probably be the first condition to be checked (a pickleader is more likely to log in than an onboarding volunteer): 

```suggestion
        group_names = [g.name  for g in self.role_groups]
        return ('pickleader' not in group_names and
            'volunteer' in group_names and
            self.password != '')
```